src/components/Table/index.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { TableWrapper, TD, TH } from './styles';
+
+const TableHeadItem = ({ item }) => <TH>{item.label}</TH>;
+
+const TableRow = ({ item, colName }) => (
+  <tr>
+    {colName.map(({ label, key, Row }) => {
+      if (Row) {
+        return <TD>{Row(label, key, item)}</TD>;
+      }
+
+      if (key.includes('.')) {
+        const keySplit = key.split('.');
+        return <TD>{item[keySplit[0]][keySplit[1]]}</TD>;
```

Comment:
if we have a more complex object like { obj: { obj:  { name: 'name'  } } }, how can we use 'name'?